typescript/infra/package.json

```diff
@@ -9,13 +9,13 @@
     "@aws-sdk/client-iam": "^3.74.0",
     "@aws-sdk/client-kms": "3.48.0",
     "@aws-sdk/client-s3": "^3.74.0",
-    "@ethersproject/experimental": "^5.3.0",
+    "@celo-tools/celo-ethers-wrapper": "^0.2.0",
+    "@ethersproject/experimental": "^5.6.2",
     "@nomiclabs/hardhat-etherscan": "^3.0.3",
     "@types/mocha": "^9.1.0",
     "@types/node": "^16.9.1",
     "@types/yargs": "^17.0.8",
     "asn1.js": "5.4.1",
-    "celo-ethers-provider": "^0.0.0",
```

Comment:
Replacing this dead dep with @celo-tools/celo-ethers-wrapper that does the same thing and that I semi-maintain